resolver.go

```diff
@@ -214,21 +240,11 @@ func resolveName(backend bind.ContractBackend, input string) (common.Address, er
 }
 
 func resolveHash(backend bind.ContractBackend, domain string) (common.Address, error) {
```

Comment:
I don't understand the purpose of the changes here.  Please don't use shortened names, it doesn't add anything (especially in a world where we have registry, registrar and resolver; r is not a useful letter).  I also don't understand why the returned error has been changed.

Reply:
renaming all the aliased variables to a more meaningful version.

the `resolver.go` had to be changed to support offchain domains the same way it does for the on-chain ones. the main goal is to maintain the syntax for both types of domain:

```go
r, _ := NewResolver(client, "onchain.eth")
actual, err := r.Text("com.twitter")

r, _ := NewResolver(client, "offchain.eth")
actual, err := r.Text("com.twitter")
```

the implementation was inspired by [the Viem's implementation](https://github.com/wevm/viem/blob/main/src/actions/ens/getEnsText.ts#L92) of the CCIP-Read which relies on the Universal Resolver instead of directly on the Resolver itself.